doc/topics/tutorials/minionfs.rst

```diff
@@ -4,10 +4,27 @@
 MinionFS Backend Walkthrough
 ============================
 
+Propagating Files
+=================
+
 .. versionadded:: 2014.1.0
 
 Sometimes, you might need to propagate files that are generated on a minion.
-Salt already has a feature to send files from a minion to the master:
+Salt already has a feature to send files from a minion to the master.
+
+Enabling File Propation
```

Comment:
Doh!
Do I need to do another PR or did someone see your comment as well and fix it ?